vision/benchmarks/accuracy/bench_module.py

```diff
@@ -339,39 +373,33 @@ def bench_module_accuracy(folder: str, quiet_output: bool = False, draw_centers:
                 # calculate execution time
                 exec_time = end_time - start_time
                 f.write(str(exec_time))
-                
+
                 total_time += exec_time
 
+                # Saves image with circles and the module in frame
+                if in_frame and save_circle:
+                    tester.accuracy_saveCircleImage(file, image, depth)
```

Comment:
This tester.accuracy... line can just be replaced with the one line from your function above.

`self.location.saveCircleImage(filename, OUTPUT_FILE)`